src/utils.js

```diff
@@ -88,7 +88,8 @@ go.utils = {
     },
 
     check_valid_alpha: function(input) {
-        var alpha_only = new RegExp('^[A-Za-z]+$');
+        // regular expression below of characters we see as invalid alphabetics; character limit of 150
+        var alpha_only = new RegExp('(^[^±!@£$%^&*_+§¡€#¢§¶•ªº«\\/<>?:;|=.,123456789]{1,150}$)');
```

Comment:
@NicolaasDieBaas I'm going to land this for now, but please open another issue and let's test the utils function seperately (I added a flow validation test)